lsp-types/src/Language/LSP/Types/DocumentColor.hs

```diff
@@ -45,10 +45,10 @@ deriveJSON lspOptions ''DocumentColorParams
 -- | Represents a color in RGBA space.
 data Color =
   Color
-    { _red   :: Int -- ^ The red component of this color in the range [0-1].
-    , _green :: Int -- ^ The green component of this color in the range [0-1].
-    , _blue  :: Int -- ^ The blue component of this color in the range [0-1].
-    , _alpha :: Int -- ^ The alpha component of this color in the range [0-1].
+    { _red   :: Float -- ^ The red component of this color in the range [0-1].
```

Comment:
Surprised to see a Float is needed for that, but i guess it is from the spec

Reply:
I think that's the best thing to correspond to a "decimal"? WDYT?

Reply:
nvm, i did not understand the usage